packages/shorebird_cli/lib/src/commands/release/release_ios_command.dart

```diff
@@ -96,6 +96,13 @@ make smaller updates to your app.
       return e.exitCode.code;
     }
 
+    if (results.rest.contains('--obfuscate')) {
+      // Obfuscated releases break patching, so we don't support them.
+      // See https://github.com/shorebirdtech/shorebird/issues/1619
+      logger.err('Shorebird does not currently support obfuscation on iOS.');
```

Comment:
I would just also point to https://github.com/shorebirdtech/shorebird/issues/1619.

Reply:
Done